### DISCUSSION TITLE
Renaming the base capacitor file from `index.html` to something else.

### DISCUSSION THREAD

**Discussion by kierangilliam** [ionic-team/capacitor#3318] [Q&A] [answered]
A framework I am using names the final distribution file `__app.html` instead of `index.html`. I tried changing the build process to spit out `index.html` but am having little success.

Can I make capacitor look for an `__app.html` file instead of the default `index.html`?

**Comment by imhoffd** [comment, marked as answer]
There is no configuration option in Capacitor for this. I would recommend looking into the configuration options of your framework to see if it's possible to use a different final distribution file. If not, you can always run a script after the web asset build to rename it.
